tests/hook/test_agent.py

```diff
@@ -62,6 +81,21 @@ def handle_start(*args, **kwargs):
             self.assertIsNotNone(inputs)
             print(f"<component> input: {inputs}")
 
+        @Hook.on_agent_action(hook_type="component")
+        def handle_action(*args, **kwargs):
+            nonlocal action_flag
+            action_flag = True
+
+            thought = kwargs["thought"]
+            action = kwargs["action"]
+            action_input = kwargs["action_input"]
```

Comment:
if kwargs["thought"] is None, this line will get error

Reply:
Optimize the test code please

Reply:
If this one doesn't have this key-value pair, then an error should be reported

Reply:
Therefore, if an error occurs, assertIsNotNone (line 91-93) is meaningless